src/instructlab/cli/data/generate.py

```diff
@@ -160,6 +161,9 @@
     type=click.IntRange(min=512),
     cls=clickext.ConfigOption,
 )
+@click.option(
+    "-dt", "--detached", is_flag=True, help="Run ilab data generate in the background"
+)
```

Comment:
Is the idea we will independently add this to each command? Should we have a goal of making this a global flag or are we only scoping process management for certain commands?

Reply:
each command, not everything will have a detached mode. Primarily any long running command but not things like `ilab model list`